data/data/bootstrap/baremetal/files/usr/local/bin/build-metal3-env.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+
+set -euo pipefail
+
+# shellcheck disable=SC1091
+source /usr/local/bin/release-image.sh
+
+export KUBECONFIG=/opt/openshift/auth/kubeconfig-loopback
+
+AUTH_DIR=/opt/metal3/auth
+set +x
+ironic_url="$(printf 'http://%s:%s@localhost:6385/v1' "$(cat "${AUTH_DIR}/ironic/username")" "$(cat "${AUTH_DIR}/ironic/password")")"
```

Comment:
This could just be templated in directly to the env file I think.

If we can avoid creating an extra service that would be good.

Reply:
The `ironic_url` yes, and I have done that.  Not sure about the podman call inside `image_for`.